charts/moco/CHANGELOG.md

```diff
@@ -14,6 +15,7 @@ If you turn off this option when updating, the CRD will be removed, causing data
 ### Changed
 - chart: add value to set a PriorityClass [#661](https://github.com/cybozu-go/moco/pull/661)
 - Helm chart CRDs optional and not deleted on uninstall [#684](https://github.com/cybozu-go/moco/pull/684), [685](https://github.com/cybozu-go/moco/pull/685)
+- Add basic Helm chart values [#687](https://github.com/cybozu-go/moco/pull/687)
 
 ### Contributors
 - @fgeorgeanybox
```

Comment:
Sorry, this line is a copy-and-paste error.
Please change to [#661](https://github.com/cybozu-go/moco/pull/661)'s author.

```suggestion
- @npdgm
```